.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Webpage
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y pandoc python3-pandocfilters make
+        pandoc --version
+        make page
+    - name: Deploy
+      if: github.repository == 'weiznich/diesel.rs-website' && github.event_name == 'push'
```

```suggestion
      if: github.repository == 'sgrif/diesel.rs-website' && github.event_name == 'push'
```

Note to myself: Change this before merging the PR.